tests/test_frechet_mean.py

```diff
@@ -149,6 +149,18 @@ def test_estimate_and_belongs_default_gradient_descent_so_matrix(self):
         expected = True
         self.assertAllClose(result, expected)
 
+    @geomstats.tests.np_and_tf_only
```

Comment:
What is the problem with pytorch?

Reply:
This test is new, it cannot be used with pytorch because the matrix log is used but not implemented in pytorch